src/compiler/typescript_to_rust/azle_generate/src/cdk_act/nodes/canister_method.rs

```diff
@@ -0,0 +1,138 @@
+use proc_macro2::{Ident, TokenStream};
+use quote::{format_ident, quote};
+
+use crate::cdk_act::{ActNode, ToTokenStream};
+
+#[derive(Clone)]
+pub enum CanisterMethodActNode {
```

Comment:
I'm thinking that CanisterMethodActNode should be within the ActNode enum, just like any of the other types of an ActNode, since CanisterMethodActNode is an ActNode

Reply:
Each ActNode variant should have other ACT nodes in them, like PrimitiveInfo I would rename to PrimitiveActNode (maybe @bdemann is taking care of this).